q2_longitudinal/tests/test_longitudinal.py

```diff
@@ -284,6 +284,28 @@ def test_linear_mixed_effects(self):
             group_categories='delivery,diet,antiexposedall',
             individual_id_column='studyid', metric='observed_otus')
 
+    def test_linear_mixed_effects_no_group_categories(self):
```

Comment:
This test isn't asserting the results are correct --- we have some examples of cracking open a viz and testing the contents in a few place (e.g. [q2-demux](https://github.com/qiime2/q2-demux/blob/master/q2_demux/tests/test_demux.py#L757-L783), [q2-diversity](https://github.com/qiime2/q2-diversity/blob/master/q2_diversity/tests/test_alpha_rarefaction.py), [q2-taxa](https://github.com/qiime2/q2-taxa/blob/master/q2_taxa/tests/test_visualizer.py)) --- I think that should be doable here to make sure the grouping working correctly.

Reply:
oops... I was being sneaky, wasn't I?

I've added `pdt.assert_frame_equal` comparisons for all LME tests now.